File: src/Application Specific/Classes/Entity Collections/IBCs/IbcSql.vb
Imports System.Collections.Generic
Imports System.Linq
Imports FSM.Entity.Sys

Namespace Entity.Ibc

    Public Class IbcSql
        Private _database As Database

        Public Sub New(ByVal database As Database)
            _database = database
        End Sub

        Public Function [GetAll]() As DataView
            _database.ClearParameter()
            Return New DataView(_database.ExecuteSP_DataTable("Ibc_GetAll"))
        End Function

    End Class

End Namespace

File: apollo/apollo.fsm/src/Application Specific/Classes/Entity Collections/IBCs/IbcSql.cs
using System.Data;
using FSM.Entity.Sys;

namespace FSM.Entity.Ibc
{
    public class IbcSql
    {
        private Database _database;

        public IbcSql(Database database)
        {
            _database = database;
        }

        public DataView GetAll()
        {
            _database.ClearParameter();
            return new DataView(_database.ExecuteSP_DataTable("Ibc_GetAll"));
        }
    }
}